src/networking.c

```diff
@@ -2229,9 +2230,14 @@ int processInlineBuffer(client *c) {
 
     /* Setup argv array on client structure */
     if (argc) {
-        if (c->argv) zfree(c->argv);
-        c->argv_len = argc;
-        c->argv = zmalloc(sizeof(robj*)*c->argv_len);
+        if (argc > CLINET_STATIC_ARGV_LEN) {
```

Comment:
```suggestion
        if (argc > CLINET_STATIC_ARGV_LEN || (c->flags & CLIENT_MULTI)) {
```

we can avoid use static argv when in multi state.
then we can remove the code above:
```
 if(c->argv == c->argv_static) {
        c->argv = zmalloc(sizeof(robj*)*c->argc);
        memcpy(c->argv, c->argv_static, sizeof(robj*)*c->argc);
        c->argv_len = c->argc;
    }
```